src/sentry/api/endpoints/codeowners/__init__.py

```diff
@@ -25,6 +25,7 @@ class ProjectCodeOwnerSerializer(CamelSnakeModelSerializer):
     raw = serializers.CharField(required=True)
     organization_integration_id = serializers.IntegerField(required=False)
     date_updated = serializers.CharField(required=False)
+    instance: ProjectCodeOwners
```

Comment:
ah, I commented on this in the other PR -- this isn't correct.  the base class is missing its generic (and would otherwise have this typed properly)

Reply:
ah yup that makes sense. will put up a fix. thanks

Reply:
i'm kind of confused at the typing stubs -- https://github.com/typeddjango/djangorestframework-stubs/blob/c78da6d315f0994ad472d2c2406ae5718ef36057/rest_framework-stubs/serializers.pyi#L195

it says instance can be a list, but in all the code i've seen of our own usage, it's always a singular model

Reply:
weird. almost looks like an oversight for `ListSerializer` above it (since `_BaseSerializer` doesn't do lists)